submissions/kolawoleOgunfowokan/HTML/js/script.js

```diff
@@ -0,0 +1,91 @@
+
+  let productArray = 
+  [
+      { 
+        image: "../images/HP250.jpg",
+        title: "HP Laptop",
+        price: "N130000"
+      },
+  
+      { 
+         image: "../images/eutetic-freezer.jpg",
+         title: "Freezer",
+         price: "N195000"
+       },
+  
+       {
+         image: "../images/huaweiMateBookXPro.jpg",
+         title: "Huwaei Laptop",
+         price: "N200000"
+       },
+  
+       {
+         image: "../images/java-prog.png",
+         title: "Java Book",
+         price: "N12000"
+       },
+  
+       {
+         image: "../images/microsoftSurface.jpg",
+         title: "Microsoft Surface",
+         price: "N320000"
+       },
+  
+       {
+         image: "../images/introprog.jpg",
+         title: "Introduction to Programming",
+         price: "N10000"
+       },
+  
+       {
+         image: "../images/Macbook.jpg",
+         title: "Macbook pro",
+         price: "N490000"
+       },
+  
+       {
+         image: "../images/AppleDesktop.png",
+         title: "Apple Desktop",
+         price: "N350000"
+       },
+  ];
+  
+  let button = document.getElementById("btn");
+  let displayContent = false;
+  let products = document.getElementById("product-box");
+    
+  button.addEventListener("click", function() {
+    displayProducts();
+  });
+  
+  function displayProducts() {
+    displayContent = !displayContent;
```

Comment:
Why toggle this at the beginning?

Reply:
Without it the product will keep populating itself. And because of the binding, without it the button refused to work.